bin/install-if-deps-outdated.js

```diff
@@ -24,7 +24,7 @@ const needsInstall = () => {
 };
 
 if ( needsInstall() ) {
-	const installResult = spawnSync( 'npm', [ 'install' ], {
+	const installResult = spawnSync( 'lerna', [ 'bootstrap', '--hoist' ], {
```

Comment:
After some DM'ing with @sirreal, I'm inclined to stick with global installation, since both installation and distclean otherwise run into nasty dependency issues that would require ugly/possibly fragile special casing to verify lerna has been installed vs (other) dependencies are outdated....